[DOCS] Update fluent In Progress Cautionary Note (#7681)
great-expectations/great_expectations: docs/docusaurus/docs/components/warnings/_in_progress.md

:::caution In progress

> Great Expectations is aware that the fluent datasource and yaml config workflows are undocumented. A new method for connecting to data sources will be added to this topic soon. In the meantime, use the [Great Expectations Slack channel](https://greatexpectations.io/slack) if you encounter issues and need assistance.

Reviewer: ```suggestion
Great Expectations is aware that the fluent Datasource and yaml config workflows are undocumented. A new method for connecting to a source data system will be added to this topic soon. In the meantime, use the [Great Expectations Slack channel](https://greatexpectations.io/slack) if you encounter issues and need assistance.
```

"Datasource" all one word is the technical term, so our convention has been to capitalize it.  To lesson confusion with "data sources" we've been using "source data systems" to refer to data as it exists outside of GX.

> There are currently broken workflows and incompatibilities with our fluent datasources and yaml config. This document is under review and a new method for connecting to data sources will be added soon. In the meantime, use the [Great Expectations Slack channel](https://greatexpectations.io/slack) if you encounter issues and need assistance.

Author: ```suggestion
Great Expectations is aware that the fluent datasource and yaml config workflows are undocumented. A new method for connecting to data sources will be added to this document soon. In the meantime, use the [Great Expectations Slack channel](https://greatexpectations.io/slack) if you encounter issues and need assistance.
```
